app/views/receptacles/new_request.html.erb

```diff
@@ -9,7 +9,7 @@
     <% instance_variable_or_id_param(:request_type) do |field_name, value| %>
       <div class="form-group  col-xs-12">
         <%= label_tag(field_name, 'Request type', class: 'col-md-4') %>
-        <%= select_field_sorted_by_name(field_name, @request_types, value, can_edit) %>
+        <%= select_field_sorted_by_name(field_name, @request_types, value, can_edit, prompt: 'Select a request type', required: true) %>
```

Comment:
Doesn't this change the default values of all fields? The story is asking for “request additional sequencing” page, the request type, read length and flowcell.